intera_interface/src/intera_interface/limb.py

```diff
@@ -146,10 +174,14 @@ def __init__(self, limb="right", synchronous_pub=False):
                    "from %s") % (self.name.capitalize(), ns + 'endpoint_state')
         intera_dataflow.wait_for(lambda: len(self._cartesian_pose.keys()) > 0,
```

Comment:
Do we still need endpoint_state if we have tip_states? Do tip_states give us everything that is in endpoint_state?